actions/checkForAndPlaceOrder.ts

```diff
@@ -29,6 +34,7 @@ import { GPv2Order } from "./types/ComposableCoW";
 import { validateOrder } from "./handlers";
 
 const GPV2SETTLEMENT = "0x9008D19f58AAbD9eD0D60971565AA8510560ab41";
+const MULTICALL3 = "0xcA11bde05977b3631167028862bE2a173976CA11";
```

Comment:
Is it the same address in all networks?

Reply:
Yep, it's the same across all networks. Their [repository refers](https://github.com/mds1/multicall).